### PR TITLE
Add train script

### PR DESCRIPTION
**Pull request by HDThang**
- Add train script for summarization task
- Add train script for text-generate task
- Add train script for token classification task
- Add common libraries for environment in env.yaml

### CODE REVIEW

**Comment by yeomni** [comment]
@HDThang Would you please resolve conflicts first?

File: examples/pytorch/text-generation/train.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+$model_type=openai-gpt
```
Comment (loctxmoreh) [inline examples/pytorch/text-generation/train.sh:3]: A minor syntax error. Please fix @HDThang

Comment (yeomni) [reply examples/pytorch/text-generation/train.sh:3]: Please note that our policy is to require at least one approval before merging into the main branch. Seems like we haven't fully reviewed these commits yet.

Comment (loctxmoreh) [reply examples/pytorch/text-generation/train.sh:3]: The title of the PR #6 said this was for merging into branch `add_mul_choice_train_script`, not `main`. Idk somehow these changes still appear on `main` :shrug:

Do you have any idea why? @HDThang @huonglarne

File: examples/pytorch/language-modeling/train_clm.sh [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-log_file="${LOG_DIR}/${model_name}.log"
-output_dir="$OUTPUT_DIR/$model_name"
+while getopts m:b:g: flag
+do
+    case "${flag}" in
+        m) model=${OPTARG};;
+        b) batch_size=${OPTARG};;
+        g) device_id=${OPTARG};;
+    esac
+done
```
Comment (loctxmoreh) [inline examples/pytorch/language-modeling/train_clm.sh:8]: While this pattern used for parsing arguments looks nice, I suggest adding default values for these args. It helps when someone just want to quickly run to verify if the script works, without fiddling with arguments.

It should be trivial. Just assign default values before this `while` block.

Comment (HDThang) [reply examples/pytorch/language-modeling/train_clm.sh:8]: Sure, I will add the default values for these arguments.

**Pull request merged**